src/cli/migrate/nodeVersion/getNodeTypesVersion.ts

```diff
@@ -28,27 +32,28 @@ export const getNodeTypesVersion = async (
       },
     });
 
-    const parsedVersion = NpmFetchResponse.safeParse(response);
-    if (!parsedVersion.success) {
-      throw new Error('Failed to parse response');
+    const parsedVersions = NpmFetchResponse.safeParse(response);
+    if (!parsedVersions.success) {
+      throw new Error('Failed to parse @types/node response from npm');
     }
 
-    const version = Object.values(parsedVersion.data)
+    const { version } = Object.values(parsedVersions.data.versions)
       .filter(
         (v) =>
           valid(v.version) &&
           satisfies(v.version, `${major}.x.x`) &&
           !v.deprecated,
       )
-      .reduce((a, b) => (gt(a.version, b.version) ? a : b)).version;
+      .reduce((a, b) => (gt(a.version, b.version) ? a : b));
 
     return {
       version,
     };
-  } catch {
+  } catch (err) {
+    log.warn(err);
     return {
       version: defaultVersion,
-      err: 'Failed to fetch latest version, using fallback version',
+      err: 'Failed to fetch latest @types/node version, using fallback version',
```

Comment:
If `defaultVersion` isn't logged elsewhere:

```suggestion
      err: `Failed to fetch latest @types/node version, using fallback version ${defaultVersion}`,
```